### ISSUE TITLE
A little confusion about training BAM.

### ISSUE THREAD

**Issue by zhouyuan888888** [chunbolang/BAM#11]
Very thanks for your great work. But I am a little confused about the details for training BAM. Have you chosen the model that achieves the best performance in the test set as the well-trained model? Does this cause a little unfairness in model evaluation? 

--Thank you so much for your reply !!!

**Comment by chunbolang** [comment]
@zhouyuan888888 
Hi, 
thanks for your interest!

The commonly used FSS datasets (pascal-5i and coco-20i) are not divided into _validation_ sets, so most advanced methods tend to save a series of checkpoints for evaluation, e.g., PMMs, PFENet, and HSNet. A straightforward trick is to save more checkpoints, while it also increases the cost of testing. 

As far as I am concerned, the above prevalent scheme is not very appropriate. It seems more reasonable to use a dataset with a validation set, such as the recently proposed FSS-1000. Anyway, as long as the test data is not leaked.

Regards,
Chunbo

**Comment by chunbolang** [comment]
You are welcome. Feel free to contact~
